picamera2/picamera2.py

```diff
@@ -839,6 +839,10 @@ def _update_libcamera_stream_config(libcamera_stream_config, stream_config, buff
         libcamera_stream_config.size = libcamera.Size(stream_config["size"][0], stream_config["size"][1])
         libcamera_stream_config.pixel_format = libcamera.PixelFormat(stream_config["format"])
         libcamera_stream_config.buffer_count = buffer_count
+        if stream_config.get("stride") is not None:
+            libcamera_stream_config.stride = stream_config["stride"]
+        else:
+            libcamera_stream_config.stride = 0
```

Comment:
Would `libcamera_stream_config.stride = stream_config.get("stride", 0)` be the same?

Reply:
Oh yes, that'll be cleaner, I'll change that and push a new commit